src/cloudfront-control-plane-stack.ts

```diff
@@ -163,7 +163,7 @@ export class CloudFrontControlPlaneStack extends Stack {
     ].join(' ');
 
     const frameCSPUrl = [
-      `*.quicksight.${Aws.PARTITION}.${Aws.URL_SUFFIX}`,
+      `*.quicksight.${Aws.PARTITION}.amazon.com`,
```

Comment:
why rollback this change? Should it always use fixed string `aws` instead of `Aws.PARTITION`?

Reply:
Let's add it to our list as a questionnaire item for the service team.